coralproject/talk
# Markdown InfoBox

## What does this PR do?
- Use [simpleMDE](https://github.com/NextStepWebs/simplemde-markdown-editor) for the InfoBox configuration.
- Render markdown into InfoBox using [marked](https://github.com/chjj/marked)

## How do I test this PR?

- Go to admin and use the MarkdownEditor to enter some content into the InfoBox.
- Go to the asset and see the markdown being rendered.
